api/tests/integration/infrastructure/repositories/authentication-method-repository_test.js

```diff
@@ -0,0 +1,94 @@
+const { expect, databaseBuilder, knex, catchErr, domainBuilder } = require('../../../test-helper');
+const authenticationMethodRepository = require('../../../../lib/infrastructure/repositories/authentication-method-repository');
+const AuthenticationMethod = require('../../../../lib/domain/models/AuthenticationMethod');
+const { AlreadyExistingEntity } = require('../../../../lib/domain/errors');
+
+describe('Integration | Repository | AuthenticationMethod', () => {
+
+  describe('#create', () => {
+
+    afterEach(() => {
+      return knex('authentication-methods').delete();
+    });
+
+    context('When creating a AuthenticationMethod of type PASSWORD', () => {
+
+      it('should return an AuthenticationMethod', async () => {
+        // given
+        const userId = databaseBuilder.factory.buildUser().id;
+        await databaseBuilder.commit();
+        const authenticationMethod = domainBuilder.buildAuthenticationMethod.buildPasswordAuthenticationMethod({
+          password: 'Password123',
+          shouldChangePassword: false,
+          userId,
+        });
+
+        // when
+        const savedAuthenticationMethod = await authenticationMethodRepository.create({ authenticationMethod });
+
+        expect(savedAuthenticationMethod).to.be.instanceOf(AuthenticationMethod);
+      });
+    });
+
+    context('When creating a AuthenticationMethod of type GAR', () => {
```

Comment:
Des deux contextes suivants, on ne pourrait en laisser qu'un seul avec le nom 
```suggestion
    context('When creating a AuthenticationMethod containing a sub', () => {
```
En prenant les suggestions pour remplacer `sub` par le terme retenu

Reply:
Done.